package.json

```diff
@@ -38,5 +38,10 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "prettier": {
```

Comment:
is this a dev dependency?

Reply:
Prettier? It should be... but its not! I'll fix it in my other PR.